lib/location/Location.ts

```diff
@@ -24,6 +24,19 @@ export type LocationCoordinate2D = Readonly<
   z.infer<typeof LocationCoordinates2DSchema>
 >
 
+/**
+ * Checks for equality on two sets of coordinates.
+ */
+export const coordinateMatch = (
```

Comment:
coordinateMatch can be taken multiple meanings, name should specify that it is checking if coords are equal